src/Symfony/Component/PropertyInfo/ConstructorArgumentTypeExtractorInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyInfo;
+
+/**
+ * Infers the constructor argument type.
+ *
+ * @author Dmitrii Poddubnyi <dpoddubny@gmail.com>
+ */
+interface ConstructorArgumentTypeExtractorInterface
```

Comment:
should be in the `Extractor` subnamespace?

Reply:
Moved. And squashed.